src/main/webapp/contact.html

```diff
@@ -1,29 +1,104 @@
+          </div>
+          <div id="podmates">
+            <div class="mate">
+              <div class="basic-info">
+                <div class="row">
+                  <img src="./assets/jpeg/uma.jpeg" alt="Picture of Uma">
+                  <div class="col">
+                    <h4>Uma Bahl | Santa Clara University | c/o 2022</h4>
+                    <div class="row" style="text-align: center;">
+                      <a href="http://github.com/ubahl/">
+                        <img src="./assets/svg/github.svg" alt="Image of the GitHub logo"> ubahl
+                      </a>
+                      <a href="https://www.linkedin.com/in/umabahl/">
+                        <img src="./assets/svg/linkedin.svg" class="svg" alt="Image of the LinkedIn logo"> umabahl
+                      </a>
+                    </div>
+                    <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                      labore et dolore magna aliqua.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="mate">
+              <div class="basic-info">
+                <div class="row">
+                  <img src="./assets/jpeg/melanie.jpeg" alt="Picture of Melanie">
+                  <div class="col">
+                    <h4>Melanie Gutzmann | Oregon State University | c/o 2022</h4>
+                    <div class="row">
+                      <a href="http://github.com/mirrorkeydev/">
+                        <img src="./assets/svg/github.svg" alt="Image of the GitHub logo"> mirrorkeydev
+                      </a>
+                      <a href="https://www.linkedin.com/in/mgutzmann/">
+                        <img src="./assets/svg/linkedin.svg" class="svg" alt="Image of the LinkedIn logo"> mgutzmann
+                      </a>
+                    </div>
+                    <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                      labore et dolore magna aliqua.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="mate">
+              <div class="basic-info">
+                <div class="row">
+                  <img src="./assets/jpeg/areeta.jpeg" alt="Picture of Areeta">
+                  <div class="col">
+                    <h4>Areeta Wong | University of California, Irvine | c/o 2022</h4>
+                    <div class="row">
+                      <a href="http://github.com/areetaw/">
+                        <img src="./assets/svg/github.svg" alt="Image of the GitHub logo"> areetaw
+                      </a>
+                      <a href="https://www.linkedin.com/in/areetaw/">
+                        <img src="./assets/svg/linkedin.svg" class="svg" alt="Image of the LinkedIn logo"> areetaw
+                      </a>
+                    </div>
```

Comment:
Would it be possible for these to wrap differently on mobile?
Instead of 
![image](https://user-images.githubusercontent.com/35010111/89955694-6dd82980-dbe8-11ea-8f00-c3ae30805ed7.png)
```text
...
c/o 2022
[logo1]      [logo2]
[username1]  [username2]
...
```
maybe
```text
...
c/o 2022
[logo1] [username1]
[logo2] [username2]
...
```
?

Reply:
Yup!